lib/solidus_segment/serializers/variant_serializer.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module SolidusSegment
+  module Serializers
+    class VariantSerializer
+      def initialize(variant)
+        @variant = variant
+      end
+
+      def to_h
+        {
+          id: variant.id,
+          sku: variant.sku,
+          name: variant.product.name,
+          variant: variant.options_text,
+          price: variant.price,
+          quantity: 1,
```

Comment:
We are not forced to set the `quantity` and also the `value` when we are serializing only the Variant (https://segment.com/docs/connections/spec/ecommerce/v2/#product-list-viewed), they are optional arguments and I think doesn't have meaning here.

Reply:
Changed